tests/structured_conf/test_structured_config.py

```diff
@@ -864,6 +864,135 @@ def test_create_untyped_list(self, class_type: str) -> None:
         assert cfg.list == [1, 2]
         assert cfg.opt_list is None
 
+    class TestInstantiateStructuredConfigs:
```

Comment:
Those tests should probably go to test_to_container()

Reply:
Done in https://github.com/omry/omegaconf/pull/502/commits/841ac01b73197e22f8195ece8c76649eb305d237